src/current/cockroachcloud/cockroachdb-advanced-on-azure.md

```diff
@@ -12,13 +12,6 @@ This page provides information about CockroachDB {{ site.data.products.dedicated
 
 CockroachDB {{ site.data.products.dedicated }} clusters on Azure have the following temporary limitations. To express interest or request more information about a given limitation, contact your Cockroach Labs account team. For more details, refer to the [FAQs](#faqs).
 
-### Editing and scaling
-
-- A cluster must have at minimum three nodes. A multi-region cluster must have at minimum three nodes per region. Single-node clusters are not supported.
-- After it is created, a cluster's storage can be increased in place, but cannot subsequently be decreased or removed.
```

Comment:
why is this deleted? Is it not true anymore?

Reply:
Because it's the same as for all clouds, and is noted in the Cluster Creation page.